### PR TITLE
Update for German locale

### PR DESCRIPTION
**Pull request by spodermenpls**
A couple of small tweaks, after seeing my translation in action the first time.

### CODE REVIEW

File: src/_locales/de/messages.yml [hunk 1]
```diff
@@ -148,7 +148,7 @@ labelAuthor:
   message: 'Autor: '
 anchorAuthor:
   description: Author shown on about tab.
-  message: '<a href=mailto:i@gerald.top>Gerald</a>'
+  message: ''
```
Comment (gera2ld) [inline src/_locales/de/messages.yml]: Why is this removed?

Comment (spodermenpls) [reply src/_locales/de/messages.yml]: Your adress and name needs no translation, and by delegating this field back to the English locale, a change there gets displayed in the German version without having to edit its locale.

Comment (gera2ld) [reply src/_locales/de/messages.yml]: OK, I am just curious since it makes no change to the result. 😄
